src/libraries/System.Private.CoreLib/src/System/Gen2GcCallback.cs

```diff
@@ -30,17 +30,17 @@ private Gen2GcCallback(Func<object, bool> callback, object targetObj)
 
         /// <summary>
         /// Schedule 'callback' to be called in the next GC.  If the callback returns true it is
-        /// rescheduled for the next Gen 2 GC.  Otherwise the callbacks stop.
+        /// rescheduled for the next Gen 2 GC.  Otherwise the callback stops.
```

Comment:
Probably should be changed to:

```suggestion
        /// rescheduled for the next Gen 2 GC, otherwise the callback stops.
```

Same for the one below. Let me know if I should make that change. I am not trying to be too picky here though, just a quick win so to speak.

Reply:
Per conversation, reverted.